frontend/src/components/TourDeFoos/TourDeFoos.tsx

```diff
@@ -36,7 +37,9 @@ function TourDeFoos() {
                             PATH_FOR_CREATING_NEW_TOURNAMENT_VIEW
                         </Route>
                         <Route path={PATH_FOR_TOURNAMENT_REGISTRATIONS_VIEW} exact>
-                            PATH_FOR_TOURNAMENT_REGISTRATIONS_VIEW
+                            <Centered>
+                                <TournamentRegistrations tournamentId="sampleTournamentId" />
```

Comment:
A pobieranie id z routera? To w kolejnym :) ?

Reply:
xd O ja, rozpędziłem się za bardzo :DD

Teraz już widze jak to działa, dzięki :D
Poprawię